backend/registration/urls.py

```diff
@@ -0,0 +1,6 @@
+from .views import RegisterAPI
+from django.urls import path
+
+urlpatterns = [
+    path('api/register/', RegisterAPI.as_view(), name='register'),
```

Comment:
Can we changed this one to `path('api/user/register', RegisterAPI.as_view(), name='register'),` to be more explicit.